tokio/src/sync/once_cell.rs

```diff
@@ -50,11 +49,17 @@ impl<T> Default for OnceCell<T> {
     }
 }
 
-impl<T> fmt::Debug for OnceCell<T> {
+impl<T: fmt::Debug> fmt::Debug for OnceCell<T> {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fmt.debug_struct("OnceCell")
-            .field("value", &self.value)
-            .finish()
+        if self.initialized() {
```

Comment:
Maybe use `OnceCell::get()` instead, to avoid unsafe block?

Reply:
Yes I could do that, but `OnceCell::get` calls the unsafe method internally. Is there something bad about having `unsafe` blocks in `Debug` impls?

Reply:
In my opinion, you should not use unsafe if it does not give you any benefits (less unsafe code - less chance for UB and less work for the reviewer). For example, it is totally fine IMHO to use unsafe code to improve performance. But I don't see any gains arising from this unsafe usage.

Anyway, it's just a minor remark, feel free to ignore.

Reply:
No, I think you're right. Not having `unsafe` code makes the code easier to read. I'll change that, thanks.